Libs/MRML/Core/vtkMRMLScene.h

```diff
@@ -162,6 +172,19 @@ class VTK_MRML_EXPORT vtkMRMLScene : public vtkObject
   /// into the already existing singleton node. That node is then returned.
   vtkMRMLNode* AddNode(vtkMRMLNode *nodeToAdd);
 
+  /// \brief Instantiate and add a node to the scene.
+  ///
+  /// This is the preferred way to create and add a new node to
+  /// the scene. It ensures that the new node properties are initialized
+  /// considering its default nodes.
```

Comment:
Add a note about unique name generation, using the name that is provided. Maybe the name argument could be called "nodeBaseName" to emphasize that this string will be used to create a unique node name in the scene.